pykechain/client.py

```diff
@@ -1556,6 +1556,43 @@ def _create_parts_bulk(
         parts = self.parts(id__in=",".join(part_ids))
         return PartSet(parts=parts)
 
+    def _delete_parts_bulk(
+            self,
+            parts: List[Union[Part2, Text]],
+            asynchronous: Optional[bool] = False,
+            **kwargs
+    ) -> bool:
+        """Delete multiple Parts simultaneously.
+
+        :param parts: list of Part objects or UUIDs
+        :type parts: List[Property] or List[UUID]
+        :param asynchronous: If true, immediately returns (default = False)
+        :type asynchronous: bool
+        :param kwargs:
+        :return: True if parts are delete successfully
+        :raises APIError: if the parts could not be deleted
+        :raises IllegalArgumentError: if there were neither Parts nor UUIDs in the list of parts
+        """
+        list_parts = list()
+        for part in parts:
+            if isinstance(part, Part2):
+                list_parts.append(part.id)
+            elif is_uuid(part):
+                list_parts.append(part)
+            else:
+                raise IllegalArgumentError("{} is not a Part nor an UUID".format(part))
+        json = {"parts": list_parts}
+        query_params = kwargs
+        query_params.update(API_EXTRA_PARAMS['parts2'])
+        query_params['async_mode'] = asynchronous
```

Comment:
What happens if `asynchronous =  "hodor"`?

Reply:
unsure. I guess 'hodor' booleanized is 'falsy'. It is indeed wise to ensure that `asynchronous` is properly sanitized as a bool. I know there are utils for that (ask @JelleBoersma )

Reply:
@JelleBoersma - could you point me to those utils @jberends is talking about?